src/components/TopBar.js

```diff
@@ -313,7 +313,7 @@ const TopBar = ({ showMenu = true, title, href, children }) => {
           div({
             style: { flex: 'none', padding: 28, marginTop: 'auto' }
           }, [
-            h(CromwellVersionLink, { variant: 'light', style: { textDecoration: 'underline', color: colors.accent(0.2) } }),
+            h(CromwellVersionLink, { variant: 'light', style: { textDecoration: 'underline', color: colors.light() } }),
```

Comment:
I made this change so that the Cromwell version text is the same color as the "Built on" text. It was just subtly different, which made contrast worse.

Reply:
Before:

![image](https://user-images.githubusercontent.com/484484/157466661-1fa9bf84-61a3-480a-a19e-a8f71023a85c.png)

After:

![image](https://user-images.githubusercontent.com/484484/157466788-321af22f-a765-4353-97fe-4dcd632becc4.png)